test/http.cc

```diff
@@ -29,6 +30,132 @@ const char* schemes[] = {"http://", "https://"};
 
 INSTANTIATE_TEST_SUITE_P(Schemes, HTTPTest, testing::ValuesIn(schemes));
 
+
+TEST_P(HTTPTest, GetSuccessASIO) {
+  std::string scheme = GetParam();
+
+  // Currently we don't provide tests for https.
```

Comment:
Can you expand the comment to explain why, and when we should be able to add https tests?